sea1/router/playbook.yaml

```diff
@@ -6,20 +7,28 @@
   become_method: doas
   tasks:
     - name: install ports
-      openbsd_pkg: name={{ item }} state=latest
+      openbsd_pkg: name={{ item }} state=present
       with_items:
         - ansible
         - git
         - htop
         - screen--
         - vim--no_x11
-      tags:
-        - slow
+    ### Tasks that require hitting the internet must go above this line ###
```

Comment:
That also means things above this line can only be changed if this role is used against the build_vm thingy, not the real router.

For my own clarification: the intent is that this role can both provision new disks (in the case of the vm thing), and update configuration on the live router (e.g. for bgpd changes). That's right, yes?

I think the readme could possibly point out that distinction, and this line can as well...

Reply:
Yes, your interpretation is right. It should probably say something closer to "tasks hitting the internet while bootstrapping must go above this line". Opened #14 :)